frontend/src/components/TicketList.js

```diff
@@ -68,9 +67,7 @@ const TicketList = ({ allTickets }) => {
                             borderRadius="0"
                             mt="0 !important"
                             onClick={() =>
-                                navigate(
-                                    `${ticket.path}?tickettypes=${tickettypes}`
-                                )
+                                preserveParamsNavigate(`${ticket.path}`)
```

Comment:
this doesn't need to be wrapped in a template string anymore

Reply:
same comment applies to the other 3 instances of redundant template strings for fi, ppr, and upr